data/json/items/melee.json

```diff
@@ -2283,5 +2283,43 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "id": "2h_flail_steel",
+    "type": "GENERIC",
+	"category": "weapons",
+    "name": "war flail",
+    "name_plural": "war flails",
+    "description": "This is a stout pole with a steel large flanged mace head on a chain atached to it, based on the peasant flail agricultural tool except now with a metal head and made to tresh people in metal armor rather than grain.",
```

Comment:
```suggestion
    "description": "This is a stout pole with a steel large flanged mace head on a chain attached to it, based on the peasant flail agricultural tool except now with a metal head and made to trash people in metal armor rather than grain.",
```